e2e/tests/stress_cpu.robot

```diff
@@ -0,0 +1,61 @@
+*** Settings ***
+Documentation    Negative Test Cases
+Resource    ../keywords/common.resource
+Resource    ../keywords/node.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/workload.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+
+Stress Volume Node CPU When Replica Is Rebuilding
+    Given Create a volume with 5 GB and 3 replicas
+    And Write data to the volume
+
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
+        When Delete replica on volume node to trigger replica rebuilding
+        And Stress the CPU of all volume nodes
+
+        Then Wait until replica on volume node rebuilt
+        And Check data is intact
+    END
+
+Stress Volume Node CPU When Volume Is Online Expanding
+    @{data_checksum_list} =    Create List
+    Set Test Variable    ${data_checksum_list}
+
+    Given Create statefulset 0 with rwo volume
+    And Write 1024 MB data to statefulset 0
+
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
+        And Stress the CPU of all volume nodes
+        When Expand statefulset 0 volume by 100 MiB
+
+        Then Wait for statefulset 0 volume size expanded
+        And Check statefulset 0 data is intact
+    END
+
+Stress Volume Node CPU When Volume Is Offline Expanding
+    @{data_checksum_list} =    Create List
+    Set Test Variable    ${data_checksum_list}
+
+    Given Create statefulset 0 with rwo volume
+    And Write 1024 MB data to statefulset 0
```

Comment:
We have Write **1024 MB** data to statefulset 0 and **Expand statefulset 0 volume by 100 MiB**

When should we use MB and MiB respectively?

Reply:
The dd command is set to use a block size of 1M, whereas the requested storage for the StatefulSet was 3Gi. We can enhance later to make them consistent, but I think its trivial.